### ISSUE TITLE
Using the /nocopyright switch produces 2 blank lines on the top of every file

### ISSUE THREAD

**Issue by matlus** [dotnet/codeformatter#102]
Using the /nocopyright switch produces 2 blank lines at the very top of every code file. I'd expect that it would not do this.

**Comment by zabulus** [comment]
Found out that the 2 blank lines is added not via CopyrightHeadersRule but NewLineAboveRule

**Comment by hickford** [comment]
This is a frustrating bug.

**Comment by jaredpar** [comment]
@hickford this should be fixed in the latest sources (I used /nocopyright on a number of projects).  Are you still seeing this with the latest sources or a particular release?

**Comment by hickford** [comment]
Hi Jared, thanks for your reply. I used https://github.com/dotnet/codeformatter/releases/tag/v1.0.0-alpha5

How do I install the latest version?

**Comment by jaredpar** [comment]
@hickford I don't believe we have a newer release.  I think we're going to have a new one shortly thougoh given all the bug fixes we made recently.
